### DISCUSSION TITLE
Dynamic Base URL for Axios

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#9587] [General - Components / Directives / etc]
Dear All , 

**Based on user preference the Base Url of Axios will get change at  run time** , for example if user select America we have to call **America** Data Center API like **https://mycompany-america.com/api** , if use select **Australia** my Base URL would be **https://mycompany-australia.com/api** , I can get user preference from Vuex store .

**Can you please help me how to change axios baseurl at runtime.**

**Comment by metalsadman** [comment]
see https://github.com/axios/axios#config-defaults, you can make a function that changes your axios's instance baseUrl.

**Reply by jmriyaz84** [reply]
Thanks for your quick help . I will try & let you know
